rollbar.js

```diff
@@ -81,7 +81,7 @@ exports.init = function (accessToken, options) {
    *  
    */
   if (!initialized) {
-    if (!accessToken) {
+    if (!accessToken && options.enabled !== false) {
```

Comment:
I'd move the `options = options || {}` above this check and add in `if (!accessToken && options.enabled)`